src/game/laramisc.c

```diff
@@ -25,6 +25,16 @@
 
 #define MOVE_TIMEOUT 90
 
+static RESUME_INFO *Lara_GetResumeInfo(int32_t level_num);
+
+static RESUME_INFO *Lara_GetResumeInfo(int32_t level_num)
+{
+    if (g_GameInfo.level_type == GFL_SAVED) {
+        return &g_GameInfo.current[level_num];
+    }
+    return &g_GameInfo.start[level_num];
+}
+
```

Comment:
`GFL_RESTART` is used indirectly here. Want to confirm this is ok. I'm not sure if the flag should change back `GFL_NORMAL` after this? Does it matter?